src/axom/sina/core/ID.hpp

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#ifndef SINA_ID_HPP
+#define SINA_ID_HPP
+
+/*!
+ ******************************************************************************
+ *
+ * \file ID.hpp
+ *
+ * \brief   Implementation file for Sina ID class
```

Comment:
Should this be "Header file" instead of "Implementation file"?

```suggestion
 * \brief   Header file for Sina ID class
```